populate/src/main/resources/seed/i18n/es/languageTexts.json

```diff
@@ -1,44 +1,43 @@
 [
-  {"keyName": "navbarDashboard", "text": "Tablero", "language": "es"},
-  {"keyName": "navbarChangePassword", "text": "Cambiar Contraseña", "language": "es"},
-  {"keyName": "navbarLogout", "text": "Cerrar Sesión", "language": "es"},
+  {"keyName": "navbarDashboard", "text": "Panel de control", "language": "es"},
```

Comment:
This file in particular is interesting to look at because it basically shows us Google Translate versus a professional translation service